### PR TITLE
[gameobj] missing skin/furrier bundles

### PR DESCRIPTION
**Pull request by mrhoribu**
Add missing bundles for #656

### CODE REVIEW

**Comment by ondreian** [comment]
I would probably add some unit tests for these, just to be sure

File: spec/gameobj-data/skin_spec.rb [hunk 1]
```diff
@@ -324,6 +321,60 @@
...
     end
 
+    describe "bundle skin oddities - recognize single skin, but not bundle" do
+      [
+        %{elongated triton spine},
+        %{iridescent triton hide},
+        %{soft red firebird feather},
+        %{curved gold-flecked claw},
+      ].each do |skin_name|
+        it "doesn't recognizes single #{skin_name} as a skin" do
```
Comment (ondreian) [inline spec/gameobj-data/skin_spec.rb:331]: shouldn't this be does recognize?

File: spec/gameobj-data/skin_spec.rb [hunk 2]
```diff
@@ -324,6 +321,60 @@
...
+          expect(skin.sellable.to_s).to eq "furrier"
+        end
+
+        pluralized_skin_name = if skin_name.end_with? "s"
+                                skin_name
+                              else
+                                "#{skin_name}s"
+                              end
+
+        it "recognizes bundle of #{pluralized_skin_name} as a skin" do
```
Comment (ondreian) [inline spec/gameobj-data/skin_spec.rb:343]: shouldn't this be doesn't?

**Review by ondreian — CHANGES_REQUESTED** [review]
i think you have 2 statements inverted, but otherwise looks good now

File: spec/gameobj-data/skin_spec.rb (hunk 1 above)
Comment (mrhoribu) [reply spec/gameobj-data/skin_spec.rb:331]: Yep, forgot to update this when copied again to do reverse lookup.

File: spec/gameobj-data/skin_spec.rb (hunk 2 above)
Comment (mrhoribu) [reply spec/gameobj-data/skin_spec.rb:343]: Yep, forgot to update this when copied again to do reverse lookup.

**Pull request merged**